src/Symfony/Component/HttpKernel/Log/DebugLoggerConfigurator.php

```diff
@@ -18,12 +18,12 @@
  */
 class DebugLoggerConfigurator
 {
-    private ?DebugLoggerInterface $processor = null;
+    private ?\Closure $processor = null;
 
-    public function __construct(DebugLoggerInterface $processor, bool $enable = null)
+    public function __construct(callable $processor, bool $enable = null)
```

Comment:
should be `ProcessorInterface|callable` for monolog 3, isn't it?

Reply:
We could change it that way, but it's not needed IMO as the `ProcessorInterface` does nothing more than enforcing the `__invoke()` method to be present: https://github.com/Seldaek/monolog/blob/main/src/Monolog/Processor/ProcessorInterface.php#L26

Reply:
oh perfect then